scripts/pipeline_node.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+import sys
+
+import rclpy
+from rclpy.node import Node
+
+from std_msgs.msg import Bool
+from geometry_msgs.msg import Pose
+from geometry_msgs.msg import PoseArray
+from pipeline_inspection.srv import GetPath
+
+
+def compare_poses(pose1, pose2, delta=0.1):
```

Comment:
here

Reply:
it could, but in this node I am not instantiating any MavrosWrapper node, and in mavroswrapper it compares with self.local.pose which is not the case here

Reply:
Ah yes, I see.